variables.tf

```
@@ -77,6 +77,12 @@ variable "log_analytics_workspace_id" {
   type        = string
 }
 
+variable "log_analytics_destination_type" {
+  description = "The type of log analytics destination to use for this Log Analytics Workspace."
+  type        = string
+  default     = "AzureDiagnostics"
```

Set default value to `null`, ref. #42:

```suggestion
  default     = null
```